### ISSUE TITLE
The configuration file contains the plaintext password in version 6.16.0

### ISSUE THREAD

**Issue by Dopmo** [TIBCOSoftware/jasperreports#232]
Hello Team,

We ran into a security scan issue -- comment contained a plaintext password.

Here is the details:
default.jasperreports.properties:
#net.sf.jasperreports.export.xls.password=pa22w0rd

We would like to delete this comment for the following reasons:
Reason 1: Even if the content is commented out, security scanning problems may still occur, causing unnecessary trouble.
Reason 2: The content in this line is commented out, indicating that it has been discarded. It is redundant and does not need to be retained.

In conclusion, We hope to delete this comment since version 6.16.0

Thanks in advance
Dong dapeng
